substrate/frame/balances/src/impl_fungible.rs

```diff
@@ -343,6 +350,20 @@ impl<T: Config<I>, I: 'static> fungible::MutateFreeze<T::AccountId> for Pallet<T
 	}
 }
 
+impl<T: Config<I>, I: 'static> HandleImbalanceDrop<T::Balance> for fungible::DecreaseIssuance<T::AccountId, Pallet<T, I>> {
```

Comment:
Either you make `fungible::DecreaseIssuance` generic over the event, or you need to recreate an implementation of `HandleImbalanceDrop` here to a new type only for `pallet-balances`.
Because this will not compile.